### PR TITLE
fix(css): update universal selector case

### LINKED ISSUE #34230: The first selector is missing with the child combinator used in the example CSS file
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#namespace_separator

### What specific section or headline is this issue about?

Namespace separator, Creating complex selectors with CSS nesting

### What information was incorrect, unhelpful, or incomplete?

In the example/playground given under the **[Namespace separator](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#namespace_separator)** and **[Creating complex selectors with CSS nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#creating_complex_selectors_with_css_nesting)** section, the last selector used in the CSS file is as below - 
```
> p {
  font-size: 1.1rem;
}
```

According to the page on [Child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator), the syntax should be of the following type.
`selector1 > selector2  { /* style properties */ }` 

Seems like selector1 is missing in the example CSS file. 

### What did you expect to see?

1. There should be a selector on the left side of the child combinator like 
```
selector1 > p {
  font-size: 1.1rem;
}
```
**OR**
2. The page on [Child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) should mention what would happen if selector1 is absent. 

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fixes https://github.com/mdn/content/issues/34230

### CODE REVIEW

**Comment by estelle** [comment]
In this case, we should have gone with option 2, "The page on [Child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) should mention what would happen if selector1 is absent." The universal selector is implied, and would not be added by a CSS developer in these cases, so should not be added in these examples. Instead, we should teach people that the universal selector is implied.

**Comment by lalaniArman** [comment]
@estelle I tried a simple example to verify if the universal selector is implied. It doesn't seem to work. Please check it out here - https://codepen.io/lalaniArman/pen/JjqpJzG 
Am I missing anything?

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ h2 {
     outline: 3px dashed red;
   }
 }
-> p {
+* > p {
```
Comment (Josh-Cena) [inline files/en-us/web/css/css_selectors/selectors_and_combinators/index.md:160]: The description says "using CSS nesting"

**Comment by Josh-Cena** [comment]
@estelle: I don't think relative selectors are valid in complex selector lists. They are only valid inside `:has`. This was what I fixed a while ago in https://github.com/mdn/content/pull/34045

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/css_selectors/selectors_and_combinators/index.md]: done

**Comment by OnkarRuikar** [comment]
> The universal selector is implied, and would not be added by a CSS developer in these cases, so should not be added in these examples. Instead, we should teach people that the universal selector is implied.

@estelle If you workout [the grammar](https://drafts.csswg.org/selectors/#grammar) from `<combinator>` you'll find that  `* > p` is a [complex-selector](https://drafts.csswg.org/selectors/#typedef-complex-selector) and `> p` is a [relative-selector](https://drafts.csswg.org/selectors/#typedef-relative-selector) and they are not the same. Relative selector works only in certain contexts like `:has(>p)`.

If you put following code in https://jigsaw.w3.org/css-validator/#validate_by_input+with_options it says `>p` is invalid:
```css
* > p {
  font-size: 1rem;
}

:has(> p) {
  font-size: 2rem;
}

> p {
  font-size: 3em;
}
```

**Review by estelle — APPROVED** [review]
In 23 year of writing CSS, I never thought to check this. 

"A child selector is made up of two or more selectors separated by ">". and while the "*" is any element, it can only be committed in "simple selectors"

Learn something new every day.

Thank you!

**Pull request merged**